src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/EndedReferralDTO.kt

```diff
@@ -21,6 +22,7 @@ class EndedReferralDTO(
         referenceNumber = referral.referenceNumber!!,
         assignedTo = referral.assignedTo?.let { AuthUserDTO.from(it) },
         referral = DraftReferralDTO.from(referral),
+        cancellationReason = referral.cancellationReason!!.id
```

Comment:
is the id the best thing to include in this response? i feel like the string representation would be more useful.

Reply:
Just to clarify. The 'id' in this case is the 3 letter string code. ('MIS') for example. The alternative would be providing the description, or both?